### ISSUE TITLE
Export/Sync DB

### ISSUE THREAD

**Issue by Megachip** [MiniKeePass/MiniKeePass#583]
Dear Community,

it seems there is only a way to copy a Passwort DB from Cloud and using it with minikeypass. But how to get this kdbx back to the cloud? (iOS 11)

**Comment by itseveryoneshub** [comment]
You have to login to the db first, then use the "Open In" icon at the bottom to export it to Dropbox or Files.

**Comment by itseveryoneshub** [comment]
On a similar note, would anyone know how to export the Key Files?

**Comment by tssmith7** [comment]
For key files you have to connect the phone to iTunes then select "File Sharing" and select the MiniKeePass app.  The list of files used by MKP will show up and you can drag a copy from iTunes to your computers file system.

**Comment by itseveryoneshub** [comment]
That worked! Thank you very much. That was the only copy I had. Is there a plan to enable the key file to be exported as well?

**Comment by Megachip** [comment]
@itseveryoneshub @tssmith7 Thx, thats a good start. But there is no sync option, right? So after every change on mac, I have to import and on every change on iOS I have to export (open in)?

**Comment by itseveryoneshub** [comment]
That is correct. The app does not sync updates automatically among devices. You are essentially downloading the dB and working locally on a copy.

**Comment by Noiwex** [comment]
That's how keepass syncing always worked. Except download/upload process can be automated so it will be fake sync.

**Comment by Megachip** [comment]
@Noiwex Keepass itself has a lot of sync possibilities. First just storing the db in the cloud or on a network drive ;) But this isn't keepass, this is MiniKeePass ;)
